packages/opentelemetry-plugin-http/src/http.ts

```diff
@@ -295,10 +308,23 @@ export class HttpPlugin extends BasePlugin<Http> {
       };
 
       return context.with(propagation.extract(headers), () => {
-        const span = plugin._startHttpSpan(
-          `${method} ${pathname}`,
-          spanOptions
-        );
+        let span: Span = new NoopSpan();
```

Comment:
Why create a noop span here? The following if/else statement will overwrite it every time.

Reply:
Typescript would report an error otherwise :/
I could ignore it with `// @ts-ignore` though

Reply:
Don't want to use a ts-ignore. It doesn't work if you just do `let span: Span`?

Reply:
It works actually, guess i tried sometime when my code wouldn't work